test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -202,6 +205,64 @@ public void createTable() throws TableExistsException, AccumuloException,
     accumuloClient.tableOperations().delete(tableName);
   }
 
+  @Test
+  public void testDefendAgainstThreadsCreateSameTableNameConcurrently()
+      throws ExecutionException, InterruptedException {
+
+    final int numTasks = 10;
+    ExecutorService pool = Executors.newFixedThreadPool(numTasks);
+
+    for (String tablename : getUniqueNames(30)) {
+      CountDownLatch startSignal = new CountDownLatch(1);
+      CountDownLatch doneSignal = new CountDownLatch(numTasks);
+      AtomicInteger numTasksRunning = new AtomicInteger(0);
+
+      List<Future<Boolean>> futureList = new ArrayList<>();
+
+      for (int j = 0; j < numTasks; j++) {
+        Future<Boolean> future = pool.submit(() -> {
+          boolean result;
+          try {
+            numTasksRunning.incrementAndGet();
+            startSignal.await();
+            accumuloClient.tableOperations().create(tablename);
+            result = true;
+          } catch (TableExistsException e) {
+            result = false;
+          } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            result = false;
```

Comment:
Seeing an interrupted exception is not an expected outcome when calling `create()` in this scenario, so should not let it pass.  Should only see the table be created successfully or a  TableExistsException being thrown.  Any other exception should be allowed to propogate and cause `future.get()` to fail which should fail the test.

```suggestion
```